src/commands/registerCommands.ts

```diff
@@ -141,9 +142,13 @@ export function registerCommands(): void {
     registerCommand('vscode-docker.volumes.prune', pruneVolumes);
     registerCommand('vscode-docker.volumes.remove', removeVolume);
 
-    registerCommand('vscode-docker.context.use', useDockerContext);
-    registerCommand('vscode-docker.context.inspect', inspectDockerContext);
-    registerCommand('vscode-docker.context.remove', removeDockerContext);
+    registerCommand('vscode-docker.contexts.configureExplorer', configureDockerContextsExplorer);
+    registerCommand('vscode-docker.contexts.help', dockerContextsHelp);
+    registerCommand('vscode-docker.contexts.inspect', inspectDockerContext);
```

Comment:
`inspect` \ `remove` \ `use` may need to do `registerWorkspaceCommand` as they result in command line calls...I think. Let's investigate and figure it out for sure.

Reply:
When I remote ssh into an Linux machine and used the list and inspect (before this change) and it uses the remote machine's context. So I guess this should be fine.

Reply:
That seems fine to me. @karolz-ms, what do you think?